pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -191,6 +191,10 @@ public class ExtensibleLoadManagerImpl implements ExtensibleLoadManager {
      * Get all the bundles that are owned by this broker.
      */
     public Set<NamespaceBundle> getOwnedServiceUnits() {
+        if (!started) {
```

Comment:
Should we consider making this value `volatile`? Because I am not sure if `getOwnedServiceUnits` will be called by another thread. :)

Reply:
Yes, using `volatile` will be better.